Client/mods/deathmatch/logic/luadefs/CLuaClientDefs.cpp

```diff
@@ -80,3 +82,20 @@ bool CLuaClientDefs::IsCapsLockEnabled()
 {
     return ((::GetKeyState(VK_CAPITAL) & 0x0001) != 0);
 }
+
+bool CLuaClientDefs::SetCursorColor(float r, float g, float b, float alpha)
+{
+    if (!g_pCore->IsMenuVisible())
```

Comment:
This function must not reject the color. See how this is implemented for `setCursorAlpha`:
```
if (!m_pCore->IsMenuVisible() && !m_pCore->GetConsole()->IsVisible())
    m_pGUI->SetCursorAlpha(fAlpha, true);
else
    m_pGUI->SetCurrentServerCursorAlpha(fAlpha);
```

Reply:
I do not get it, it just updating the value of alpha member variable which make no sense if we checking if the menu is not visible why we upding the alpha value 

```
void CGUI_Impl::SetCurrentServerCursorAlpha(float fAlpha)
{
    m_fCurrentServerCursorAlpha = fAlpha;
}
```